cmd/describeaccount_test.go

```diff
@@ -226,13 +226,16 @@ func TestDescribeAccount_Callout(t *testing.T) {
 
 	_, uPK, _ := CreateUserKey(t)
 	_, aPK, _ := CreateAccountKey(t)
+	_, xPK, _ := CreateNkey(t, nkeys.PrefixByteCurve)
```

Comment:
CreateCurveKey()

Reply:
Now that the nkey api correctly dispatches to the right API this is safe.